packages/perseus-editor/src/widgets/expression-editor.tsx

```diff
@@ -426,7 +422,7 @@ class ExpressionEditor extends React.Component<any, any> {
         e,
     ) => {
         const newProps: Record<string, any> = {};
-        newProps.functions = _.compact(e.target.value.split(/[ ,]+/));
+        newProps.functions = e.target.value.split(/[ ,]+/).filter(Boolean);
```

Comment:
`.filter(Boolean)` is not correct in the new Typescript world (see https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/2201682693/TypeScript+for+Flow+Developers#Predicate-Functions-vs-Type-Predicates). 

```suggestion
        newProps.functions = e.target.value.split(/[ ,]+/).filter(isTruthy /* from wonder-stuff */);
```